cypress/integration/Modal.spec.ts

```diff
@@ -409,22 +408,19 @@ describe('Modal', () => {
           });
         });
 
-        it('should transfer focus to the header element', () => {
-          cy.findByRole('dialog', {name: 'Delete Item'})
-            .pipe(h.modal.getTitle)
+        it('should transfer focus to the cancel button', () => {
+          cy.findByRole('button', {name: 'Cancel'})
             .should('have.focus');
         });
 
         it('should trap focus inside the modal element', () => {
-          cy.findByRole('dialog', {name: 'Delete Item'})
-            .pipe(h.modal.getTitle)
```

Comment:
This should probably be put back into the test but in a different order since the order is now changed with setting `ref` on the Cancel Button. 

```tsx
        it('should trap focus inside the modal element', () => {
          cy.findByRole('button', {name: 'Cancel'})
            .should('have.focus');
          cy.tab()
            .should('contain', 'Delete')
            .tab()
            .pipe(h.modal.getTitle)
            .should('have.focus')
            .tab()
            .should('contain', 'Cancel')
          cy.focused().should('have.text', 'Cancel');
        });
```